analysis/R/decode.R

```diff
@@ -276,6 +279,46 @@ Resample <- function(e) {
   list(estimates = estimates, stds = stds)
 }
 
+# Private function
+# Decode for Boolean RAPPOR inputs
+# Returns a list with attribute fit only. (Inference and other aspects
+# currently not incorporated because they're unnecessary for association.)
+.DecodeBoolean <- function(counts, params, num_reports) {
+  # Set number of cohorts to 1
+  params$m = 1
+  # Sums across cohorts.
+  summed_counts = colSums(counts)
+
+  # With params$m = 1, EstimateBloomCounts computes Boolean RAPPOR estimates
+  # given counts summed across the cohorts.
+  es <- EstimateBloomCounts(params, summed_counts)
+
+  fit = as.data.frame(matrix(ncol = 7, nrow = 2))
+  colnames(fit) = c("string", "estimate", "std_error", "proportion",
+               "prop_std_error", "prop_low_95", "prop_high_95")
+
+  fit$string <- c("TRUE", "FALSE")
```

Comment:
It would be nicer if this were c(TRUE, FALSE) -- booleans not strings.  But I suspect this is something that has to be changed all over the code.